backend/controllers/users.js

```diff
@@ -9,6 +9,63 @@ const {
   Word,
 } = require("../models");
 
+const _getActivities = async (idList) => {
+  return await Activity_log.findAll({
+    where: { user_id: idList },
+    include: [
+      {
+        model: User_follow,
+        include: [
+          {
+            model: User,
+            as: "Follower",
+            attributes: ["first_name", "last_name"],
+          },
+          {
+            model: User,
+            as: "Following",
+            attributes: ["first_name", "last_name"],
+          },
+        ],
+      },
+      {
+        model: User,
+        attributes: ["first_name", "last_name"],
+      },
+      {
+        model: Lesson,
+        attributes: ["id", "title"],
+        include: [
+          {
+            model: Lesson_word,
+          },
+        ],
+      },
+    ],
+  });
+};
+
+const _addLessonScore = async (activity_logs) => {
+  return await Promise.all(
+    activity_logs.map(async (al) => {
+      if (al.relatable_type === "lesson") {
+        const newAl = JSON.parse(JSON.stringify(al));
```

Comment:
- choose proper naming of variables. `newAl` does not mean something. make it readable and verbose as possible (ATR)